docs/examples/101/cosmosdb-webapp/main.bicep

```diff
@@ -84,7 +84,7 @@ resource website 'Microsoft.Web/sites@2020-06-01' = {
         }
         {
           name: 'CosmosDb:Key'
-          value: listKeys(cosmosAccount.id, cosmosAccount.apiVersion).primaryMasterKey
+          value: cosmosAccount.listKeys().primaryMasterKey
```

Comment:
I'll probably revert this - was using it for testing. I don't think we should show off syntax that hasn't yet been released, as the example won't work.

Reply:
Yeah good pt.